nixos/modules/services/web-apps/mobilizon.nix

```diff
@@ -0,0 +1,353 @@
+{ pkgs, lib, config, ... }:
+
+with lib;
+
+let
+  cfg = config.services.mobilizon;
+
+  user = "mobilizon";
+  group = "mobilizon";
+
+  settingsFormat = pkgs.formats.elixirConf { };
+
+  configFile = settingsFormat.generate "mobilizon-config.exs" cfg.settings;
+
+  # Make a package containing launchers with the correct envirenment, instead of
+  # setting it systemd services, so that the user can also use them without troubles
```

Comment:
instead of setting it WITH systemd services (is what I think you meant)

So the idea behind the laucher is that the user can start the program on their own, correct? I don't have an opinion on that per se, but do you foresee a use case where the user will have to do this?
I thought that the purpose of the service was to abstract that away from the user. So that he would only have to do systemctl start/stop. Just curious if you had something in mind that I don't see here.

Reply:
Just an additional thought here, won't they be needing RELEASE_TMP in order to run it themselves?

Reply:
> instead of setting it WITH systemd services (is what I think you meant)

:+1: 

> So the idea behind the laucher is that the user can start the program on their own, correct? I don't have an opinion on that per se, but do you foresee a use case where the user will have to do this?
> I thought that the purpose of the service was to abstract that away from the user. So that he would only have to do systemctl start/stop. Just curious if you had something in mind that I don't see here.

These launchers are both used in the systemd service, and installed in the environment, but the reason I put them in the environment is not for the user to start it by themselves, but because the `mobilizon` command provides a `remote` subcommand which allows the user to drop an Elixir shell into a running Mobilizon instance, which is very useful for debugging / interacting with Mobilizon (in fact, I already used it several types to look at the effective configuration). It also provides the `mobilizon_ctl`  command which allows the user to rollback database migrations, and manage users in case the instance is invite only, among [other things](https://docs.joinmobilizon.org/administration/CLI%20tasks/).


> Just an additional thought here, won't they be needing RELEASE_TMP in order to run it themselves?

Good catch, it's probably better to set it here

Reply:
Great, I think it would be nice to add why the user would want to run it themselves, it might not be straightforward for everybody. What you are saying makes perfect sense.